charon/src/translate_functions_to_ullbc.rs

```diff
@@ -1887,14 +1883,12 @@ impl<'tcx, 'ctx> TransCtx<'tcx, 'ctx> {
         let generics = bt_ctx.get_generics();
         let preds = bt_ctx.get_predicates();
 
-        let body = if rust_id.is_local() && is_transparent {
-            // It's a local and transparent global: we extract its body as for functions.
-            match bt_ctx.translate_body(rust_id.expect_local(), 0) {
-                Err(_) => {
-                    // Error case: we could have a specific variant
-                    None
-                }
-                Ok(body) => Some(body),
+        let body = if is_transparent {
```

Comment:
Maybe add a comment here to explain that we test whether the definition is local or not in `translate_body`(I was quite puzzled when reviewing this change at first)

Reply:
We could do the `is_transparent` check in `translate_body` instead?

Reply:
I did that, lmk if you like it